docs/asyncapi.yaml

```diff
@@ -196,6 +206,21 @@ components:
           $ref: '#/components/schemas/TransactionDate'
         type:
           $ref: '#/components/schemas/TypeDax'
+    D365:
+      type: object
+      properties:
+        paymentReference:
+          $ref: '#/components/schemas/PaymentReference'
+        calculationReference:
```

Comment:
is your d365 database instance not calculationId, not calculationReference?

Reply:
No, everything has to be mapped from ending with Id to ending with Reference